packages/server-core/src/hooks/send-invite.ts

```diff
@@ -42,10 +42,21 @@ import { Application } from '../../declarations'
 import logger from '../ServerLogger'
 import config from '../appconfig'
 import { InviteParams } from '../social/invite/invite.class'
-import { getInviteLink, sendEmail, sendSms } from '../user/auth-management/auth-management.utils'
 
 const emailAccountTemplatesPath = path.join(appRootPath.path, 'packages', 'server-core', 'email-templates', 'invite')
 
+/**
+ * A method which get an invite link
+ *
+ * @param type
+ * @param id of accept invite
+ * @param passcode
+ * @returns invite link
+ */
+export function getInviteLink(type: string, id: string, passcode: string): string {
+  return `${config.server.url}/a-i/${id}?t=${passcode}`
```

Comment:
As we've merged in the accept-invite conversion, this should now be using acceptInvitePath instead of 'a-i'.

Reply:
Done